src/ManagedServices/ManagedServices/Commands/NewAzureRmManagedServicesDefinition.cs

```diff
@@ -32,10 +32,14 @@ public class NewAzureRmManagedServicesDefinition : ManagedServicesCmdletBase
         protected const string DefaultParameterSet = "Default";
         protected const string ByPlanParameterSet = "ByPlan";
 
-        [Parameter(ParameterSetName = DefaultParameterSet, Mandatory = true, HelpMessage = "The name of the Registration Definition.")]
```

Comment:
making mandatory parameters optional are breaking changes and are not allowed in non-breaking change release